trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0...10]
+  end
```

Comment:
- [ ]  Why dont you use the if condition here since this method is to trim the word If it's longer? I suggest that you first check if the word has a maximum of 10 characters and if so then it trims the word. 😊 I will also suggest you check if it is 0 to 9, not 10, remember it is index based.